mfr/extensions/jasp/templates/viewer.mako

```diff
@@ -0,0 +1,9 @@
+<link rel="stylesheet" href="/static/css/bootstrap.min.css">
```

Comment:
not sure. i think all the other renderers have this, and i assumed it was necessary, but don't really know what it does.